tests/console/git.pm

```diff
@@ -42,6 +42,7 @@ sub run {
     type_string("yes\n");
     assert_screen("password-prompt");
     type_string("$password\n");
+    assert_screen 'root-console';
```

Comment:
why you not using here testapi function type_password here ?

Reply:
Well, should not change the behaviour we want to fix here but true, we should use `type_password`. @Soulofdestiny while you are at the module, could you please simplify it by removing line L21 and replacing `type_string(password)` with `type_password`? Could be a second commit in the same PR.

Reply:
but it would need to be

```perl
type_password;
send_key 'ret';
```

Reply:
changed - retriggered verification run

Reply:
make sure to update the code then as well